test/utils/TestUtil.scala

```diff
@@ -47,8 +47,7 @@ trait TestUtil extends UnitSpec with GuiceOneAppPerSuite with BeforeAndAfterEach
 
   lazy val fakeRequestWithVrnAndRedirectUrl: FakeRequest[AnyContentAsEmpty.type] =
     FakeRequest().withSession(
-      SessionKeys.clientVrn -> vrn,
-      SessionKeys.redirectUrl -> "/homepage"
+      SessionKeys.clientVrn -> vrn
```

Comment:
This fake request is now identical to the one below it (just has the one session key which is the client's VRN). Could you delete one of them and make sure that any tests that used it now use the other one (and rename it if it mentions the redirect URL in the name).